README.md

```diff
@@ -83,6 +91,20 @@ The following word should always be capitalized:
  - Keywi
  - Keepass
  - Secure Storage
+
+### Translation of addons.mozilla.org metadata
+ 1. fork this repo and clone it into your workspace
+ 2. copy the `_amo/en/description.html` file to e.g. `_amo/nl/description.html`
+ 2. copy the `_amo/en/summary.html` file to e.g. `_amo/nl/summary.html`
+ 3. translate it
+ 4. commit and push to Github
+ 5. open a pull request
+
+In `description.html` these HTML tags are supported:
```

Comment:
What about `summary.html`, maybe explicit list that one as well, or just refer to 'the html files'?

Reply:
The summary actually only supports text, I updated the README